xchange-bithumb/src/test/resources/org/knowm/xchange/bithumb/dto/account/example-order.json

```diff
@@ -1,10 +1,10 @@
 {
-  "order_id": "1546705688665840",
+  "order_id": "C01040000000000000000",
   "order_currency": "XRP",
   "order_date": "1546705688665840",
   "payment_currency": "KRW",
   "type": "ask",
-  "status": "placed",
+  "status": "0000",
```

Comment:
is this actually something they are returning?

in the documentation I can only see:  send，pending，success，cancel

Reply:
"status": "0000" is the top level (did the rest call work)
"order_status": "Completed" is see on orderDetails
send，pending，success，cancel i dont see in docs

it is clear that the code we had was written for an API that changed drastically, but remained on the same endpoints

Reply:
ok